tests/python/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+SET(${PROJECT_NAME}_PYTHON_TESTS
+  test_talos_walk_sl1m_quasistatic
+  test_talos_walk_sl1m_quasistatic_no_store
+  test_talos_walk_rbprm_mopt
+  test_talos_walk_sl1m_mopt_dyn_filter
+  test_talos_walk_sl1m_topt
+  test_talos_stairs_manual_limb_rrt
+  test_talos_stairs_sl1m_mopt
+  )
+
+FOREACH(TEST ${${PROJECT_NAME}_PYTHON_TESTS})
+  ADD_PYTHON_UNIT_TEST("py-${TEST}" "tests/python/${TEST}.py")
+  SET_TESTS_PROPERTIES("py-${TEST}" PROPERTIES RUN_SERIAL "ON" TIMEOUT 600)
```

Comment:
You should add a comment about this `RUN_SERIAL` here, as this is something we don't usually find in our CMakeLists, and people might wonder what it is.